### ISSUE TITLE
Max data disks per node for E80ids_v4

### ISSUE THREAD

**Issue by kholger** [NetAppDocs/cloud-volumes-ontap#213]
Page: [Storage limits for Cloud Volumes ONTAP 9.10.1 in Azure](https://docs.netapp.com/us-en/cloud-volumes-ontap/reference_limits_azure_9101.html)

Hi, 

In the section for storage limits for 9.10.1 there is a limit for instance type E80ids_v4 specified with 61 disks.
For 9.10.0 and earlier, there is a limit with 29 disks.
https://docs.netapp.com/us-en/cloud-volumes-ontap/reference_limits_azure_9100.html#single-node-with-capacity-based-licensing

Based on the feedback from PM, the disk limit should be independent from ONTAP release. 

Please verify which number is correct and adjust if required.

Thank you!
Best
Holger

**Comment by netapp-bcammett** [comment]
Hi Holger,

Thank you for reaching out. Yes, it should be 61 disks across all releases. I'm in the process of making the required doc update.

Thanks again,
Ben

**Comment by kholger** [comment]
Hi Ben,

Great, perfect.
Thank you very much.

Best
Holger
________________________________

**Comment by kholger** [comment]
Looking at the Microsoft docs, it even shows max 64 data disks:
https://docs.microsoft.com/en-us/azure/virtual-machines/edv4-edsv4-series

Size
vCPU
Memory: GiB
Temp storage (SSD) GiB
Max data disks
Max temp storage throughput: IOPS/MBps*
Max uncached disk throughput: IOPS/MBps
Max burst uncached disk throughput: IOPS/MBps1
Max NICs
Max network bandwidth (Mbps)
Standard_E80ids_v4 3
80
504
2400
64
375000/4000
80000/1200
80000/2000
8
30000

Thx
Holger

Von: Kaercher, Holger ***@***.***>
Datum: Mittwoch, 6. April 2022 um 19:43
An: NetAppDocs/cloud-volumes-ontap ***@***.***>, NetAppDocs/cloud-volumes-ontap ***@***.***>
Cc: Author ***@***.***>
Betreff: Re: [NetAppDocs/cloud-volumes-ontap] Max data disks per node for E80ids_v4 (Issue #213)
Hi Ben,

Great, perfect.
Thank you very much.

Best
Holger
________________________________

**Comment by netapp-bcammett** [comment]
Yes, I am confirming with engineering though, because our internal engineering docs list 32 max data disks (which is really 29 when you take away three for internal disks).
